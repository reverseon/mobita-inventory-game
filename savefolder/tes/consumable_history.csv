id;id_pengambil;id_consumable;tanggal_peminjaman;jumlah
120;75844;C20;23/10/2001;20
121;75844;C5;23/10/2004;21
122;75844;C20;23/10/2001;22
123;75844;C5;23/10/2004;23
124;75844;C20;23/10/2001;24
125;75844;C5;23/10/2004;25
126;75844;C20;23/10/2001;26
127;75844;C5;23/10/2004;27
