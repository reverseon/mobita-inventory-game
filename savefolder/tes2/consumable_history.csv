id;id_pengambil;id_consumable;tanggal_peminjaman;jumlah
120;75844;C20;23/10/2001;20
