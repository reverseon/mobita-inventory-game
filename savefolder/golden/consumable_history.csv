id;id_pengambil;id_consumable;tanggal_pengambilan;jumlah
100147;38070;C4;12/07/2020;20
19331;53615;C3;04/11/2013;20
